src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+
+const ChatLog = (props) => {
+// props is going to an array of objects
+    const chatEntryArray = props.entries.map((chatEntry) => {
+        return (
+            <ChatEntry
+                key={chatEntry.id}
+                id = {chatEntry.id}
+                sender={chatEntry.sender}
+                body={chatEntry.body}
+                timeStamp={chatEntry.timeStamp}
+                liked={chatEntry.liked}
+                onUpdate={props.onUpdate}
```

Comment:
You can install a *linter* in your IDE that will format your space, indentation, etc **on save**. Consider installing ESLint and/or Prettier to help with code formatting.